sensor/common/compliance/service_impl.go

```diff
@@ -149,15 +149,14 @@ func (s *serviceImpl) startSendingLoop(stopper concurrency.Stopper) {
 					}
 				})
 			} else {
-				con, ok := s.connectionManager.connectionMap[msg.Hostname]
-				if !ok {
+				con, found := s.connectionManager.connectionMap[msg.Hostname]
+				if found {
+					err := con.Send(msg.Msg)
+					if err != nil {
+						log.Errorf("Error sending MessageToComplianceWithAddress to node %q: %v", msg.Hostname, err)
+					}
+				} else {
```

Comment:
won't block the PR over this, but usually I've seen `if/else` discouraged in favor of using a `return` to minimize indentation

Reply:
Yeah, I prefer that too. Let me see if that can be improved here.